xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampToken.xml

```diff
@@ -240,11 +283,31 @@ The value from calling <xref:System.Security.Cryptography.Pkcs.SignedCms.Encode?
       </Parameters>
       <Docs>
         <param name="signerInfo">To be added.</param>
-        <param name="signerCertificate">To be added.</param>
-        <param name="extraCandidates">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="signerCertificate">When this method returns, the certificate from the Timestamp Authority (TSA) which signed this token, or <see langword="null"/> if a signer certificate cannot be determined. This parameter is treated as uninitialized.</param>
+        <param name="extraCandidates">An optional collection of certificates to consider as the Timestamp Authority (TSA) certificates, in addition to any certificates that may be included within the token.</param>
+        <summary>Verifies that the current token is a valid time-stamp token for the provided <see cref="T:System.Security.Cryptography.Pkcs.SignerInfo"/>.</summary>
+        <returns><see langword="true"/> if the Timestamp Authority (TSA) certificate was found, the certificate public key validates the token signature, and the token matches the signature for <paramref name="signerInfo"/>; otherwise, <see langword="false"/>.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+This method verifies that the time-stamp token is valid for a Cryptographic Message Syntax (CMS) <xref:System.Security.Cryptography.Pkcs.SignerInfo>, according to IETF RFC 3161 Appendix A (Signature Time-stamp attribute using CMS).
+
+This method requires that the Timestamp Authority (TSA) certificate has the required extended key usage value (1.3.6.1.5.5.7.3.8), matches the token's embedded ESSCertID or ESSCertID2 identifier, has a validity range that encompasses the token's timestamp value, and has a public key that successfully validates the token's signature.
+
+After determining the appropriate TSA certificate, this method determines if the signature value from the provided SignerInfo matches the digest and algorithm identifier from the token.
+If the SignerInfo signature matches the token's digest and algorithm identifier, the certificate is reported via the `signerCertificate` and the method returns `true`.
+
+If the SignerInfo signature does not match, or the TSA certificate could not be determined, the method reports `signerCertificate` as `null` and returns `false`.
```

Comment:
Since there is a parameter called `signedInfo`, the string SignerInfo should probably be formatted to point to it:

```suggestion
If the `signerInfo` signature does not match, or the TSA certificate could not be determined, the method reports `signerCertificate` as `null` and returns `false`.
```

Reply:
Personally, I thought it looked better as the type (hence "SignerInfo"); but I'm open to changing it.